### PR TITLE
Issue #393: Fix rpmverifyfile compilation on RHEL5

### PR DESCRIPTION
**Pull request by jan-cerny**
One missing include and some fallback definition of macros
will fix the issue.

### CODE REVIEW

File: src/OVAL/probes/unix/linux/rpmverifyfile.c [hunk 1]
```diff
@@ -76,6 +76,18 @@ struct rpmverify_res {
 #define RPMVERIFY_SKIP_GHOST  0x2000000000000000
 #define RPMVERIFY_RPMATTRMASK 0x00000000ffffffff
 
+#ifndef RPMVERIFY_FILEDIGEST
```
Comment (ybznek) [inline src/OVAL/probes/unix/linux/rpmverifyfile.c]: I think, you cannot check enum value in the #ifdef, it will be always not defined.

**Comment by ybznek** [comment]
LGTM, I will wait until test results

**Comment by ybznek** [comment]
test this please, again

File: configure.ac [hunk 2]
```diff
@@ -413,6 +413,11 @@ PKG_CHECK_MODULES([rpm], [rpm >= 4.6],[
 ],[
 	AC_MSG_NOTICE([libprm is older than 4.6])
 ])
+PKG_CHECK_MODULES([rpm], [rpm >= 4.7],[
+	AC_DEFINE([HAVE_RPM47], [1], [Define to 1 if rpm is newer than 4.7.])
+],[
+	AC_MSG_NOTICE([libprm is older than 4.7])
```
Comment (mpreisler) [inline configure.ac]: libprm?

Comment (jan-cerny) [inline configure.ac]: Ooops. Thank you!

**Pull request merged**